stock_barcodes_supplierinfo/wizard/stock_barcodes_read.py

```diff
@@ -1,21 +1,23 @@
+# Copyright 2022 NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
You are mixing LGPL and AGPL. I think the module manifest is LGPL. Could you modify ?